hdbt.theme

```diff
@@ -102,6 +105,21 @@ function hdbt_preprocess_html(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function hdbt_preprocess_media(&$variables) {
+  if (empty($variables['media']) || !$variables['media'] instanceof EntityInterface) {
+    return;
+  }
+  if ($variables['media']->bundle() === 'remote_video') {
+    $variables['privacy_policy_url'] = _hdbt_eu_cookie_compliance_get_privacy_policy_url();
+    // @todo Get correct video service URL once multiple services are supported.
+    $variables['video_service_url'] = 'youtube.com';
```

Comment:
This should be something more generic, like `service_url` since we might want to use this same variable for other bundles as well (in future).